woocommerce/woocommerce-android
# [Deposit Summary] Learn more click handling. No deposits handling

Closes: #10070
Closes: #10135


### Description

The PR adds
* Handling of the learn more click
* Hide the deposit section if there are no last or paid deposits in the response

### Testing instructions

* More -> Payments -> Expand -> learn more -> notice chrome tab
* Using either preview or site without last and paid deposits notice that the deposits section is not shown anymore

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/9aeb5b98-629d-447a-b222-2258511cc6b5
